packages/components/src/templates/next/components/complex/Hero/Hero.tsx

```diff
@@ -27,17 +27,17 @@ const HeroGradient = ({
       }}
     >
       <div
-        className={`w-full content-end ${
+        className={`w-full content-center ${
           alignment === "left" ? "bg-gradient-to-r" : "bg-gradient-to-l"
-        } from-[rgba(0,0,0,85%)] md:from-[rgba(0,0,0,75%)] xl:from-[rgba(0,0,0,65%)]`}
+        } from-[rgba(0,0,0,95%)] to-[rgba(0,0,0,70%)] md:from-[rgba(0,0,0,75%)] xl:from-[rgba(0,0,0,95%)]`}
```

Comment:
i thought we are using hsl?

Reply:
not needed until we need branding colours!

this is just for the gradient, where rgba is sufficient